src/fides/api/ops/api/v1/endpoints/user_endpoints.py

```diff
@@ -113,6 +118,32 @@ def update_user_password(
     return current_user
 
 
+@router.post(
+    urls.USER_FORCE_PASSWORD_RESET,
+    dependencies=[Security(verify_oauth_client, scopes=[USER_PASSWORD_RESET])],
+    status_code=HTTP_200_OK,
+)
+def update_other_user_password(
+    *,
+    db: Session = Depends(deps.get_db),
+    user_id: str,
+    data: UserForcePasswordReset,
+) -> FidesUser:
+    """
+    Update a user's password given a `user_id`.
```

Comment:
Please can you add more context to the docstring here — it's important to mention why this exists separately to the endpoint above.